src/course-home/outline-tab/SequenceLink.jsx

```diff
@@ -84,42 +94,44 @@ const SequenceLink = ({
   );
 
   return (
-    <li>
-      <div className={classNames('', { 'mt-2 pt-2 border-top border-light': !first })}>
-        <div className="row w-100 m-0">
-          <div className="col-auto p-0">
-            {complete ? (
-              <FontAwesomeIcon
-                icon={fasCheckCircle}
-                fixedWidth
-                className="float-left text-success mt-1"
-                aria-hidden="true"
-                title={intl.formatMessage(messages.completedAssignment)}
-              />
-            ) : (
-              <FontAwesomeIcon
-                icon={farCheckCircle}
-                fixedWidth
-                className="float-left text-gray-400 mt-1"
-                aria-hidden="true"
-                title={intl.formatMessage(messages.incompleteAssignment)}
-              />
-            )}
-          </div>
-          <div className="col-10 p-0 ml-3 text-break">
-            <span className="align-middle">{displayTitle}</span>
-            <span className="sr-only">
-              , {intl.formatMessage(complete ? messages.completedAssignment : messages.incompleteAssignment)}
-            </span>
-            <EffortEstimate className="ml-3 align-middle" block={sequence} />
-          </div>
+    <li className={classNames({ 'active-sequence': isActive, 'border-top border-light': !first })}>
+      <div className="row w-100 m-0 d-flex align-items-center">
+        <div className="col-auto p-0" style={{ fontSize: '18px' }}>
```

Comment:
[question]: Could you move `fontSize: '18px'` to SCSS + convert to REM

Reply:
All changes in `src/course-home/outline-tab` will be reverted in next PR: https://github.com/raccoongang/frontend-app-learning/pull/17